README.md

```diff
@@ -255,10 +258,10 @@ implements `context` for options which you can import to make existing
 [`anyhow`]: https://github.com/dtolnay/anyhow
 [`tracing_error::SpanTrace`]: https://docs.rs/tracing-error/*/tracing_error/struct.SpanTrace.html
 [`stable-eyre`]: https://github.com/eyre-rs/stable-eyre
-[`color-eyre`]: https://github.com/eyre-rs/color-eyre
+[`color-eyre`]: https://github.com/eyre-rs/eyre/tree/master/color-eyre
 [`jane-eyre`]: https://github.com/yaahc/jane-eyre
 [`simple-eyre`]: https://github.com/eyre-rs/simple-eyre
-[`color-spantrace`]: https://github.com/eyre-rs/color-spantrace
+[`color-spantrace`]: https://github.com/eyre-rs/eyre/tree/master/color-spantrace
```

Comment:
LTGM -- I was worried for a moment that this was changing from a page including the README.md to one that did not but github is smart enough that it's not a concern